src/main/python/view/dataset.py

```diff
@@ -168,6 +168,26 @@ def on_clicked_delete_images_button(self):
             self._reload_images(self.__selected_dataset_category())
 
     def on_clicked_train_button(self):
+
+        if not [img for img in os.listdir(Dataset.images_path(Dataset.Category.TEST_NG)) if
+                     Path(img).suffix in ['.jpg', '.jpeg', '.png', '.gif', '.bmp']]:
```

Comment:
@akshuga 
`['.jpg', '.jpeg', '.png', '.gif', '.bmp']` should be reused as a local variable in lines 173, 179 and 185.

Reply:
add local variable "img_suffix_list" in on_clicked_train_button